src/lib/remark-plugins/remark-plugin-adjust-link-urls/helpers.ts

```diff
@@ -94,10 +123,16 @@ const preAdjustUrl = ({ currentPath, url }): string => {
 		return url
 	}
 
-	// Do nothing if url is not an internal path
-	const isInternalPath = getIsInternalPath(url)
```

Comment:
At this point have we already weeded out external links?

Reply:
The `try/catch` that is replacing this call to `getIsInternalPath` is functionally the same as the one in `getIsInternalPath`. In the `try`, if `new URL(url)` doesn't throw an error, then we know we're looking at an external link. So we check the `hostname` and only pre-adjust the URL if the `hostname` is `developer.hashicorp.com`. Otherwise, we don't do anything in the `catch` and continue on in the function because it's an internal url.

Reply:
The `handleFullUrl` added in https://github.com/hashicorp/dev-portal/pull/1544/commits/b0ba9d6a739e48aed9b7ff17b137c4afda1a616e should help bring clarity here as well!

Reply:
Those comments are great! Thank you 😄